src/H5VLconnector.h

```diff
@@ -445,9 +445,10 @@ typedef struct H5VL_token_class_t {
 //! [H5VL_class_t_snip]
 typedef struct H5VL_class_t {
     /* Overall connector fields & callbacks */
-    unsigned int       version;          /**< VOL connector class struct version #     */
+    unsigned           version;          /**< VOL connector class struct version #     */
     H5VL_class_value_t value;            /**< Value to identify connector              */
     const char *       name;             /**< Connector name (MUST be unique!)         */
+    unsigned           conn_version;     /**< Version # of connector                   */
```

Comment:
also if we have that conn_version field I guess it could be helpful to be able to query it / print it from the HDF5 public API. It would be an easy way of doing sanity checks to make sure the right version of a plugin is loaded.

Reply:
We chatted about adding the connector info field to the struct in the async channel this morning.   The library ignores it currently, so it's merely informative, but another API to get info about a connector would be good in the future, maybe even a small command-line tool.